x-pack/test/apm_api_integration/tests/service_nodes/get_service_nodes.spec.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import url from 'url';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import archives from '../../common/fixtures/es_archiver/archives_metadata';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const registry = getService('registry');
+  const supertest = getService('legacySupertestAsApmReadUser');
+
+  const archiveName = 'apm_8.0.0';
+  const { start, end } = archives[archiveName];
```

Comment:
If it's not too difficult it would be great if you can replace this with `synthtrace` which is our new data generator tool ([docs](https://github.com/elastic/kibana/blob/main/packages/elastic-apm-synthtrace/README.md))

```suggestion
  const archiveName = 'apm_8.0.0';
  const { start, end } = archives[archiveName];
```
```suggestion
  const synthtraceEsClient = getService('synthtraceEsClient');
  
  const start = new Date('2021-01-01T00:00:00.000Z').getTime();
  const end = new Date('2021-01-01T00:15:00.000Z').getTime() - 1;  
```

Reply:
Also, see an example here https://github.com/elastic/kibana/blob/main/x-pack/test/apm_api_integration/tests/dependencies/metadata.spec.ts#L14